### PR TITLE
libmypaint: new recipe

### PR DESCRIPTION
**Pull request by wattoc**

### CODE REVIEW

**Comment by wattoc** [comment]
Build server does not like the ISC licence?

**Comment by wattoc** [comment]
Still WIP

File: media-libs/libmypaint/libmypaint-1.6.1.recipe [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+number of other painting programs."
+HOMEPAGE="https://github.com/mypaint/libmypaint"
+COPYRIGHT="2008-2011 Martin Renold and contributors"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://github.com/mypaint/libmypaint/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9d888a0f13deeba06f2d63949426f4a5e4656f451b9052e4d8f7fd372a6f274e"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```
Comment (Begasus) [inline media-libs/libmypaint/libmypaint-1.6.1.recipe:12]: Doesn't build with gcc2 (no libjson_c there), builds ok with gcc8
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

File: media-libs/libmypaint/libmypaint-1.6.1.recipe [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:intltool_update$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python
```
Comment (Begasus) [inline media-libs/libmypaint/libmypaint-1.6.1.recipe:52]: python2.7 is deprecated, use python3.7 (or higher)
```diff
- 	cmd:python
+ 	cmd:python3.7
```

File: media-libs/libmypaint/libmypaint-1.6.1.recipe [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	devel:libglib_2.0$secondaryArchSuffix
+#	devel:libgirepository_1.0$secondaryArchSuffix
+	devel:libjson_c$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:intltool_update$secondaryArchSuffix
```
Comment (Begasus) [inline media-libs/libmypaint/libmypaint-1.6.1.recipe:48]: No secondaryArchSuffix needed here:
```diff
- 	cmd:intltool_update$secondaryArchSuffix
+ 	cmd:intltool_update
```

**Review by Begasus — CHANGES_REQUESTED** [review]
Some changes needed for 32bit

**Review by Begasus — APPROVED** [review]

**Comment by Begasus** [comment]
Thanks! :+1:

**Pull request merged**